tox.ini

```diff
@@ -7,7 +7,7 @@ skipsdist = True
 setenv =
   VIRTUAL_ENV={envdir}
   PYTHONWARNINGS=default::DeprecationWarning,ignore:The distutils package is deprecated,ignore:The distutils.sysconfig module is deprecated,ignore:the imp module is deprecated,ignore:SelectableGroups dict interface is deprecated
-usedevelop = False
+usedevelop = True
```

Comment:
It should not matter now that `mock_run_cmd_and_forward_stdout` is applied right?